lib/php/rendition-factory/src/Transformer/Video/ModuleCommonArgsDTO.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Alchemy\RenditionFactory\Transformer\Video;
+
+use Alchemy\RenditionFactory\Config\ModuleOptionsResolver;
+use Alchemy\RenditionFactory\Context\TransformationContextInterface;
+use Alchemy\RenditionFactory\Transformer\Video\FFMpeg\Format\FormatInterface;
+use FFMpeg;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+class ModuleCommonArgsDTO
```

Comment:
DTO is a readonly object only transferring information, not business logic.
You can either remove the `DTO` from the name of this class or move the logic from the constructor to the `AbstractVideoTransformerBase` class (better solution IMO)